### PR TITLE
adjusted width and horizontal scroll bar

### LINKED ISSUE #1572: [Add] Fixing page width and allignments
### Details

i will be adjusting page widths

### Screenshots

image

### Type of Contribution

- [ ] Update to an existing Animated Button
- [ ] Adding a new Button
- [X] Resolving a bug
- [ ] Proposal to the Repository
- [ ] Changes related to documentation or README.md
- [ ] Other Changes

### Checklist

- [X] I have checked the existing [issues](https://github.com/Spyware007/Animating-Buttons/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/Spyware007/Animating-Buttons/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Swapnil-Singh-99**
## Fixes Issue

Closes #1572 

## Changes proposed

I have removed horizontal scroll bar and fixed page width of all the pages

## Screenshots

image
image

## Checklist

- [x] You've linked this PR to the correct issue.
- [x] You have checked that the code is working correctly.
- [x] You ⭐️ the repository!

## Note to reviewers

Please merge this PR under GSSOC'23

### CODE REVIEW

**Comment by adity2911** [comment]
Good job
Resolve conflicts as well.

**Comment by Swapnil-Singh-99** [comment]
@adity2911 Please review the changes , I have resolved conflicts

File: src/components/Navbar/Navbar.module.css [hunk 1]
```diff
@@ -123,21 +124,11 @@
   text-shadow: 0px 4px 4px 0px #00000040;
   filter: drop-shadow(0px 4px 1px rgba(255, 255, 255, 0.25));
 }
-.navbar.navbar-light .github .username {
```
Comment (adity2911) [inline src/components/Navbar/Navbar.module.css:126]: Why this change?

**Comment by Swapnil-Singh-99** [comment]
@adity2911 Please review the changes

**Comment by adity2911** [comment]
can you resolve the conflicts?
Also kindly explain what changes are made!

**Comment by adity2911** [comment]
Also, the width of the page had been fixed by someone else.

**Comment by Swapnil-Singh-99** [comment]
> can you resolve the conflicts? Also kindly explain what changes are made!

Yes I have resolved the conflicts 
Code Explained:
The heading for button section has been given some top margin
I also made changes for the site width , which as said by you has already been fixed by someone

You can merge this incase you find it ok

**Comment by Swapnil-Singh-99** [comment]
I have also rebased the code with new commits in the main branch
